README.md

```diff
@@ -49,6 +49,13 @@ sudo apt install make gcc openjdk-8-jdk pkg-config meson zip \
                  libsdl2-dev
 ```
 
+For Fedora you need to install the following packages:
+```bash
+# enable RPM fusion free
+sudo dnf install https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm https://download1.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm
```

Comment:
I have no experience with Fedora, but I have some questions.

> [RPM Fusion](https://rpmfusion.org/) provides software that the Fedora Project or Red Hat doesn't want to ship.

Which packages require RPM fusion?

Among them, which ones require the non-free repo?

Reply:
ffms2-devel which contains libavformat

Reply:
OK. It should be in the free repo, so the nonfree is not necessary, is it?

Reply:
ah sorry, yes. I removed the non-free part!
https://www.rpmfind.net/linux/rpm2html/search.php?query=ffms2-devel